lib/ecto/migrator.ex

```diff
@@ -397,8 +397,10 @@ defmodule Ecto.Migrator do
       See `c:Ecto.Repo.put_dynamic_repo/1`.
 
   """
-  @spec run(Ecto.Repo.t, binary | [{integer, module}], atom, Keyword.t) :: [integer]
+  @spec run(Ecto.Repo.t, [String.t] | [{integer, module}], atom, Keyword.t) :: [integer]
   def run(repo, migration_source, direction, opts) do
+    migration_source = List.wrap(migration_source)
```

Comment:
```suggestion
    migration_sources = List.wrap(migration_source)
```

?

Reply:
yeah, happy to change that too. My understanding was that before the source was a path or a list of tuples, so now it's always a list so it didn't change that much after all. wdyt?

Reply:
Sounds good then!